docs/tutorial.md

```diff
@@ -0,0 +1,369 @@
+
+```
+        if (resource.getStatus() == null
+                || !CollectionUtils.isEqualCollection(podNames, resource.getStatus().getNodes())) {
+            if (resource.getStatus() == null) resource.setStatus(new MemcachedStatus());
+            resource.getStatus().setNodes(podNames);
+            return UpdateControl.updateStatusSubResource(resource);
+        }
+```
+
+### createMemcachedDeployment
+
+Thie method simply creates the Deployment as specified below.
+
+```
+    private Deployment createMemcachedDeployment(Memcached m) {
+        return new DeploymentBuilder()
+            .withMetadata(
+                new ObjectMetaBuilder()
+                    .withName(m.getMetadata().getName())
+                    .withNamespace(m.getMetadata().getNamespace())
+                    .withOwnerReferences(
+                        new OwnerReferenceBuilder()
+                            .withApiVersion("v1alpha1")
+                            .withKind("Memcached")
+                            .withName(m.getMetadata().getName())
+                            .withUid(m.getMetadata().getUid())
+                            .build())
+                    .build())
+            .withSpec(
+                new DeploymentSpecBuilder()
+                    .withReplicas(m.getSpec().getSize())
+                    .withSelector(
+                        new LabelSelectorBuilder().withMatchLabels(labelsForMemcached(m)).build())
+                    .withTemplate(
+                        new PodTemplateSpecBuilder()
+                            .withMetadata(
+                                new ObjectMetaBuilder().withLabels(labelsForMemcached(m)).build())
+                            .withSpec(
+                                new PodSpecBuilder()
+                                    .withContainers(
+                                        new ContainerBuilder()
+                                            .withImage("memcached:1.4.36-alpine")
+                                            .withName("memcached")
+                                            .withCommand("memcached", "-m=64", "-o", "modern", "-v")
+                                            .withPorts(
+                                                new ContainerPortBuilder()
+                                                    .withContainerPort(11211)
+                                                    .withName("memcached")
+                                                    .build())
+                                            .build())
+                                    .build())
+                            .build())
+                    .build())
+            .build();
+    }
+```
+
+### deleteResource
+
+The `deleteResource` method is an implemented method. The deletion part will be taken care of by the Java Operator SDK  library, that's why it is empty.
+
+## Run the Operator Locally
+
+### 1. Run locally outside the cluster
```

Comment:
Why number it? there are no other steps below. 
```suggestion
### Run locally outside the cluster
```

Reply:
Because 2nd number will be for running it within cluster by creating container image.

Reply:
I will modify it for now.